gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Project.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.gcloud.spi.ResourceManagerRpc.Permission;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A Google Cloud Resource Manager project object.
+ *
+ * This class' member variables are immutable.  Methods that change or update the underlying Project
+ * information return a new Project instance.
+ */
+public class Project {
```

Comment:
You can copy a good description from [here](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects#Project).